src/components/utrecht/Select.test.tsx

```diff
@@ -0,0 +1,408 @@
+import { render, screen } from "@testing-library/react";
+import { createRef } from "react";
+import { Select, SelectOption } from "./Select";
+import "@testing-library/jest-dom";
+
+describe("Select", () => {
+  it("renders a combobox role element", () => {
+    render(<Select />);
+
+    const select = screen.getByRole("combobox");
+
+    expect(select).toBeInTheDocument();
+  });
+
+  it("renders an HTML select element", () => {
+    const { container } = render(<Select />);
+
+    const select = container.querySelector("select:only-child");
+
+    expect(select).toBeInTheDocument();
+  });
+
+  it("displays as CSS inline-block element", () => {
+    const { container } = render(<Select />);
+
+    const select = container.querySelector(":only-child");
+
+    expect(select).toBeVisible();
+    expect(select).toHaveStyle({ display: "inline-block" });
+  });
+
+  it("renders a design system BEM block class name", () => {
+    const { container } = render(<Select />);
+
+    const select = container.querySelector(":only-child");
+
+    expect(select).toHaveClass("utrecht-select");
+  });
+
+  it("renders options", () => {
+    render(
+      <Select>
+        <SelectOption value="a">Option A</SelectOption>
+        <SelectOption value="b">Option B</SelectOption>
+      </Select>
+    );
+
+    const select = screen.getByRole("combobox");
+
+    const option = screen.getByRole("option", {
+      name: "Option B",
+    });
+
+    expect(select).toContain(option);
+  });
+
+  describe("invalid variant", () => {
+    it("can have an invalid state", () => {
+      const { container } = render(<Select invalid />);
+
+      const select = container.querySelector(":only-child");
+
+      expect(select).toBeInTheDocument();
+    });
+
+    it("is not invalid by default", () => {
+      const { container } = render(<Select />);
+
+      const select = container.querySelector(":only-child");
+
+      expect(select).not.toBeInvalid();
+    });
+
+    it("renders a design system BEM modifier class name", () => {
+      const { container } = render(<Select invalid />);
+
+      const select = container.querySelector(":only-child");
+
+      expect(select).toHaveClass("utrecht-select--invalid");
```

Comment:
Should we also double check the modifier className is not set when the component is not invalid?

Reply:
Maybe one test to validate no modifier classes are set by default?